queries/buildings.jinja2

```diff
@@ -73,6 +79,8 @@ SELECT
     NULL AS roof_orientation,
     "addr:housenumber" AS addr_housenumber,
     "addr:street" AS addr_street,
+    tags->'addr:floor' AS addr_floor,
+    tags->'level' as level,
```

Comment:
Usually its not a problem until it is-large apartment towers were suddenly there are too many address points to display.

If this introduces too much cost to the query we should drop it for now, was trying to edge case / future proof.

_n

Sent from my handsful device.



> On Sep 25, 2015, at 04:04, Matt Amos <notifications@github.com> wrote:
> 
> In queries/buildings.jinja2:
> 
> > @@ -73,6 +79,8 @@ SELECT
> >      NULL AS roof_orientation,
> >      "addr:housenumber" AS addr_housenumber,
> >      "addr:street" AS addr_street,
> > +    tags->'addr:floor' AS addr_floor,
> > +    tags->'level' as level,
> Taginfo suggests that only 0.02% of nodes tagged with addr:housenumber also have level set. Were there some specific nodes you were looking at where level was important?
> 
> ―
> Reply to this email directly or view it on GitHub.
>